prow/plugins/bugzilla/bugzilla.go

```diff
@@ -312,6 +323,16 @@ Once a valid bug is referenced in the title of this pull request, request a bug
 		needsValidLabel, needsInvalidLabel = valid, !valid
 		if valid {
 			log.Debug("Valid bug found.")
+			// if configured, move the bug to the new state
+			if options.StatusAfterValidation != nil {
+				if err := bc.UpdateBug(e.bugId, bugzilla.BugUpdate{Status: *options.StatusAfterValidation}); err != nil {
+					log.WithError(err).Warn("Unexpected error udpdating Bugzilla bug.")
+					return comment(fmt.Sprintf(`An error was encountered updating the bug to the %s state on the Bugzilla server at %s for bug %d:
+> %v
+Please contact an administrator to resolve this issue, then request a bug refresh with <code>/bugzilla refresh</code>.`,
+						*options.StatusAfterValidation, bc.Endpoint(), e.bugId, err))
+				}
+			}
```

Comment:
Is there a reason why we shouldn't leave a comment in GH about the bug status change on success?

Reply:
Good call, will do